### PR TITLE
Add comment field into the model

### LINKED ISSUE #145: Add COMMENTS to UML diagram, Java class and test file

### PR DESCRIPTION
**Pull request by Exeexe93**
Add comment field into the model.
Add test cases for affected commands, parsers and java files.
Modify the Ui for displaying the comment.

### CODE REVIEW

File: src/main/java/seedu/address/model/comment/Comment.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    /*
+     * The first character of the address must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    public static final String VALIDATION_REGEX = "[^\\s].*";
+
+    public final String commentMade;
+
+    /**
+     * Constructs a {@code Tag}.
```
Comment (JeremyLoh) [inline src/main/java/seedu/address/model/comment/Comment.java]: Should be `{@code Comment}` here

Comment (Exeexe93) [reply src/main/java/seedu/address/model/comment/Comment.java]: Thanks for the lookout! Corrected.

**Review by JeremyLoh — APPROVED** [review]
Looks good to merge. Great job!

**Review by khsc96 — APPROVED** [review]
LGTM

**Pull request merged**